.github/workflows/browser-compatibility-test.yml

```diff
@@ -0,0 +1,417 @@
+name: Browser Compatibility Test
+
+on:
+  schedule:
+    # More information on cron https://crontab.guru/
+    - cron: '0 0 * * *'
+  pull_request:
+      branches:
+        - master
+
+env:
+  SELENIUM_GRID_PROVIDER: saucelabs
+  CLOUD_WATCH_METRIC: false
+  TEST_TYPE: Browser-Compatibility-Test
+  SAUCE_USERNAME: ${{secrets.SAUCE_USERNAME}}
+  SAUCE_ACCESS_KEY: ${{secrets.SAUCE_ACCESS_KEY}}
+  MESSAGING_USER_ARN: ${{secrets.MESSAGING_USER_ARN}}
+  SLACK_JS_SDK_DEV_CORE_WEBHOOK: ${{secrets.SLACK_JS_SDK_DEV_CORE_WEBHOOK}}
+
+jobs:
+  browser-compatibility-audio:
+    name: Audio Compatibility Test
+    runs-on: ubuntu-latest
+    outputs:
+      job-status: ${{ job.status }}
+
+    steps:
+      - name: Checkout Package
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Pack the Chime SDK and install the tarball into the Demo
+        run: |
+          current_version=$(.github/script/get-current-version)
+          echo "Packing current version:" $current-version
+          npm run build
+          npm pack
+          cd demos/browser
+          npm uninstall amazon-chime-sdk-js
+          npm install ../../amazon-chime-sdk-js-$current_version.tgz
+      - name: Create a Job ID
+        id: create-job-id
+        uses: filipstefansson/uuid-action@ce29ebbb0981ac2448c2e406e848bfaa30ddf04c
+      - name: Set JOB_ID Env Variable
+        run: echo "JOB_ID=${{ steps.create-job-id.outputs.uuid }}" >> $GITHUB_ENV
+      - name: Echo Job ID
+        run: echo "${{ steps.create-job-id.outputs.uuid }}"
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
+      - name: Setup Sauce Connect
+        uses: saucelabs/sauce-connect-action@v1
+        with:
+          username: ${{ secrets.SAUCE_USERNAME }}
+          accessKey: ${{ secrets.SAUCE_ACCESS_KEY }}
+          noSSLBumpDomains: all
+          tunnelIdentifier: ${{ steps.create-job-id.outputs.uuid }}
+      - name: Setup Node.js - 14.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14.x
+      - name: Install Kite
+        run: integration/js/script/install-kite
+      - name: Clean Install
+        run: npm ci
+      - name: Run Audio Compatibility Test
+        run: npm run test:browser-compatibility-audio
+  browser-compatibility-video:
+    name: Video Compatibility Test
+    runs-on: ubuntu-latest
+    outputs:
+      job-status: ${{ job.status }}
+
+    steps:
+      - name: Checkout Package
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Pack the Chime SDK and install the tarball into the Demo
```

Comment:
Wonder if we could make this into an action in the future so we don't have to duplicate all this code for every test.

Reply:
I think we can solve this problem via composite actions. That should be a separate PR.

Reply:
Yeah no need to do it now. Just create a SIM for it maybe :D